platform/linux-generic/include/odp_queue_scalable_internal.h

```diff
@@ -35,6 +35,7 @@ struct queue_entry_s {
 	sched_elem_t     sched_elem;
 
 	odp_ticketlock_t ODP_ALIGNED_CACHE lock;
+	odp_atomic_u64_t timers;
```

Comment:
Same comment here as @muvarov mentioned earlier. `timers` here is too vague.  Something to indicate this is a count of the number of active timers would be better.